Description: if we fail to pin rootfs, continue to start container
 If it fails it's likely a readonly fs, in which case the reason
 for pinning (preventing remounting readonly) is moot.
Author: Serge Hallyn <serge.hallyn@ubuntu.com>
Forwarded: yes
Index: lxc-0.9.0/src/lxc/start.c
===================================================================
--- lxc-0.9.0.orig/src/lxc/start.c	2013-06-05 21:30:08.000000000 +0200
+++ lxc-0.9.0/src/lxc/start.c	2013-06-05 21:32:20.200439112 +0200
@@ -821,8 +821,7 @@
 
 	handler->pinfd = pin_rootfs(handler->conf->rootfs.path);
 	if (handler->pinfd == -1) {
-		ERROR("failed to pin the container's rootfs");
-		goto out_delete_net;
+		INFO("failed to pin the container's rootfs, (proceeding)");
 	}
 
 	/* Create a process in a new set of namespaces */
